articles/140_topic_and_service_name_mapping.md

```diff
@@ -243,41 +221,72 @@ Here is a non-exhaustive list of prefixes:
 | ROS Parameter        | rp     |
 | ROS Action           | ra     |
 
-While all planned prefixes consist of two characters, i.e. `rX`, anything proceeding the first namespace separator, i.e. `__`, can be considered part of the prefix.
+While all planned prefixes consist of two characters, i.e. `rX`, anything proceeding the first namespace separator, i.e. `/`, can be considered part of the prefix.
```

Comment:
I think it's fine as-is, because a DDS observer would see `['rt/my_topic/is/'] + cool` (`['partition1', ...] + topic_name`), and so "anything proceeding the first namespace separator" would just be `rt`. However, if you want to explain why this is ok, i.e. because of the truncating there is no leading `/`, then that's fine too.